### ISSUE TITLE
ASUS Zephyr G14, 2024 - GA403UI - LID0 state no longer being reported correctly in v41

### ISSUE THREAD

**Issue by daegalus** [ublue-os/bluefin#2041]
### Describe the bug

LID0 state is not updatd when I close the lid with an extenral monitor plugged in. It is always `open`. This prevents the display from properly disabling and causes window management inconveniences and unecessary rendering/processing of extra screen space that isn't used or visible while the lid is closed.

```
cat /proc/acpi/button/lid/LID0/state 
state:      open
```

### What did you expect to happen?

With an external monitor plugged in, I close the lid, and the internal display disables, leaving me with just the external display.

### Output of `bootc status`

```shell
❯ sudo bootc status
[sudo] password for yulian: 
No staged image present
Current booted state is native ostree
Current rollback state is native ostree
```

### Output of `groups`

```shell
❯ groups
yulian wheel lxd incus-admin docker libvirt
```

### Extra information or context

Since upgrading from Project Bluefin v40 to v41, I no longer get proper updating of the LID0 status on my Asus laptop.

I believe this is the change from using the Asus Kernel to the Bazzite kernel between the 2 versions. The Asus Kernel worked great and the LID0 was properly being updated.

The result of this is when I have an external monitor plugged in and closed the lid, it does not disable the laptop screen, and in turn it maintains desktop on it. For a while I thought I broke my setup when I upgraded because the laptop screen was set to default and all my apps, and taskbars were on that screen. Later realized everything was on there and the screen was not being managed properly.

I was originally on Project Bluefin 40, as I moved from Bazzite for the Asus Kernel support (I had issues with Bazzite not supporting many fixes needed for my laptop, that hadn't hit upstream yet.) After upgrading to 41, the LID issue started. Happens on Bazzite or Bluefin (makes sense since they use the same kernel now.)

I currently on `bazzite-gnome-asus-nvidia-open:stable` but this started with Bluefin on `bluefin-dx-asus-nvidia:latest`. I am currently rebasing to `bazzite-gnome-asus-nvidia-open:latest` to see if there might be an update there to make a difference.

**Comment by castrojo** [comment]
Yeah it's a kernel issue, we need some help either sorting out those patches or helping people spin up some extra images: https://universal-blue.discourse.group/t/something-happen-to-the-asus-image-and-kernel/5484/5

**Comment by castrojo** [comment]
https://github.com/ublue-os/bluefin/issues/2051
